src/standalone/run.go

```diff
@@ -11,10 +11,10 @@ import (
 )
 
 const (
-	noHostTenant = "-"
+	NoHostTenant = "-"
```

Comment:
I think the constants in this package should be moved to `config.go`. As it is now that file only contains the logger and having the constants in a central location would be nice.